tests/phpunit/tests/multisite/networkQuery.php

```diff
@@ -8,6 +8,8 @@
 	 * @group ms-network
 	 * @group ms-network-query
 	 * @group multisite
+	 *
+	 * @covers WP_Network_Query::query
```

Comment:
```suggestion
	 * @coversDefaultClass WP_Network_Query
```

Reply:
The methods should have `@covers ::query`

Reply:
as all test are the same function I thick do a single @covers on the class is clearer 
if we had different functions from the same class then your change would work